### PR TITLE
WebXR API update

### PR DESCRIPTION
**Pull request by modulesio**
Goal: make the latest versions of https://immersive-web.github.io/webxr-samples/ work again.

Fixes https://github.com/exokitxr/exokit/issues/747.

### CODE REVIEW

**Comment by modulesio** [comment]
Initial working:

Capture 27

File: src/XR.js [hunk 1]
```diff
@@ -356,7 +326,7 @@ class XRWebGLLayer {
     this.alpha = alpha;
     this.multiview = multiview;
```
Comment (chrisplatorres) [inline src/XR.js]: this can be removed per https://github.com/immersive-web/webxr/commit/bb483c993df49e94c3660e6a7b046885d1570779

Comment (modulesio) [reply src/XR.js]: Thanks!

**Comment by modulesio** [comment]
I think this should be good -- the WebXR samples are working provided we force WebXR entry. Reality tabs and other sites work the same.

- [ ] might want to add an entry button hack for immersive web samples + A-Frame, for testing
- [ ] the rest of examples need to be updated to the new API

**Comment by todrobbins** [comment]
giphy

**Pull request merged**